dist/index.html

```diff
@@ -1,42 +1,65 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" name="description" content="A website that collects recipes to eventually cook and store in a pantry that is full of ingredients.">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Fresh Picks</title>
-    <link rel="stylesheet" type="text/css" href="../styles.css">
+    <link rel="stylesheet" type="text/scss" href="./css/index.scss">
     <link href="https://fonts.googleapis.com/css?family=Quicksand:400,600,700&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Sulphur+Point:400,700&display=swap" rel="stylesheet">
-    <link rel="icon" href="./images/apple-logo.png"/>
-    <link rel="icon" href="./images/search.png"/>
-    <link rel="icon" href="./images/cookbook.png"/>;
-    <link rel="icon" href="./images/seasoning.png"/>;
-    <link rel="icon" href="./images/apple-logo-outline.png"/>
-    <link rel="icon" href="./images/menu.png"/>
+    <link rel="icon" href="./images/apple-logo.png" alt="image of an apple"/>
+    <link rel="icon" href="./images/search.png" alt="image of a magnifying glass"/>
+    <link rel="icon" href="./images/cookbook.png" alt="image of a cookbook"/>
+    <link rel="icon" href="./images/seasoning.png" alt="image of a list"/>
+    <link rel="icon" href="./images/apple-logo-outline.png" alt="outline image of an apple"/>
+    <link rel="icon" href="./images/shopping-cart-icon.png" alt="image of a shopping cart"/>
+    <link rel="icon" href='./images/down-arrow.png' alt="down arrow menu icon"/>
   </head>
-  <body>
+  <body role="main" aria-label="">
     <header>
       <h1><span>fresh</span>Picks<img src="./images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
       <form id="search" role="search">
-        <input id="search-input" type="text" autocomplete="off" placeholder="Search Recipes..."></input>
-        <label for="search-input" class="search-label">Search</label>
+        <label id="search-area" for="search-input" class="search-label">
```

Comment:
Whatever is happening here is probably the problem with the search bar looking weird for you.